w10h01/test/pgdp/trials/TrialOfTheDreamsTest.java

```diff
@@ -0,0 +1,87 @@
+package pgdp.trials;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.function.Function;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TrialOfTheDreamsTest {
+    private Function<byte[], Boolean> getLock() {
+        return (byte[] bytes) -> {
+            if (bytes.length != combination.length) {return false;}
```

Comment:
`combination` does not exist in current context

Reply:
damn it, you are rigth....